demo/MainWindow.cpp

```diff
@@ -419,10 +419,10 @@ void MainWindowPrivate::createContent()
 	TitleBar->insertWidget(Index + 1, CustomButton);
 	QObject::connect(CustomButton, &QToolButton::clicked, [=]()
 	{
-		auto DockWidget = createEditorWidget();
-		DockWidget->setFeature(ads::CDockWidget::DockWidgetDeleteOnClose, true);
-		DockManager->addDockWidgetTabToArea(DockWidget, DockArea);
-		_this->connect(DockWidget, SIGNAL(closeRequested()), SLOT(onEditorCloseRequested()));
+		auto clicked_DockWidget = createEditorWidget();
+		clicked_DockWidget->setFeature(ads::CDockWidget::DockWidgetDeleteOnClose, true);
+		DockManager->addDockWidgetTabToArea(clicked_DockWidget, DockArea);
+		_this->connect(clicked_DockWidget, SIGNAL(closeRequested()), SLOT(onEditorCloseRequested()));
```

Comment:
I would leave the variable as it was, it doesn't match the coding standard of this repo.